src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java

```diff
@@ -280,6 +285,26 @@ void shouldAssignCaseToApplicantSolicitorOneAndRemoveCreator1v2DSUnregisteredRes
         }
     }
 
+    @Test
+    void handleEventsReturnsTheExpectedCallbackEvents() {
+        assertThat(assignCaseToUserHandler.handledEvents()).contains(ASSIGN_CASE_TO_APPLICANT_SOLICITOR1,
+                                                                     ASSIGN_CASE_TO_APPLICANT_SOLICITOR1_SPEC);
+    }
+
+    @Test
+    void shouldReturnUnSpecCamundaTask_whenUnSpecEvent() {
+        assertThat(assignCaseToUserHandler.camundaActivityId(CallbackParamsBuilder.builder()
+                                                                 .request(CallbackRequest.builder().eventId(
+            "ASSIGN_CASE_TO_APPLICANT_SOLICITOR1").build()).build())).isEqualTo(TASK_ID);
+    }
+
+    @Test
+    void shouldReturnSpecCamundaTask_whenSpecEvent() {
+        assertThat(assignCaseToUserHandler.camundaActivityId(CallbackParamsBuilder.builder()
+                                                                 .request(CallbackRequest.builder().eventId(
+            "ASSIGN_CASE_TO_APPLICANT_SOLICITOR1_SPEC").build()).build())).isEqualTo(TASK_ID_SPEC);
+    }
```

Comment:
Add a test similar to these ones that captures the exception being thrown on line 70 of AssigneCaseToUserHandler, so we get to 100% line coverage.

Reply:
Done in @ParameterizedTest. Resolving conversation.